extensions/mssql/src/objectExplorerNodeProvider/objectExplorerNodeProvider.ts

```diff
@@ -139,6 +139,14 @@ export class MssqlObjectExplorerNodeProvider extends ProviderBase implements sql
 				let children = await node.getChildren(true);
 				if (children) {
 					expandResult.nodes = children.map(c => c.getNodeInfo());
+					// There is only child returned when failure happens
+					if (children.length === 1) {
+						let child = children[0].getNodeInfo();
+						if (child && child.nodeType === constants.MssqlClusterItems.Message) {
```

Comment:
Can we just rewrite this so instead of adding the Message node, we return an error message? I was working around the lack of the behavior you've just added here.